src/deploy-actions.js

```diff
@@ -38,11 +38,10 @@ const filterableItems = ['apis', 'triggers', 'rules', 'dependencies', ...package
  * @param {boolean} [options.actionCode] param: if false, skips action code deployment
  * @returns {Promise<object>} deployedEntities
  */
-async function deployActions (config, deployConfig = {}, logFunc, options) {
+async function deployActions (config, deployConfig = {}, logFunc = console.log, options = { actionCode: true }) {
```

Comment:
We are calling `deployActions` from app `deploy`command, where `options` could not be an empty object since we are creating it just above using it : `const options = { actionCode: flags['action-code'] }`
So, I am not sure if this case would arise: `deployActions (config, {}, null, {})`. Please correct me if I am wrong. @shazron

Reply:
@moritzraho Since Shaz is not here for the week to review this conversation, what do you think on it?

Reply:
I +1 Shaz's suggestion to set `actionCode = true` default option

Reply:
Thanks @moritzraho for the comments. I made suggested changes to the default param in the APIs. Can you please review these changes once and approve the PR if everything seems alright.